src/Parser/Visitor/ExtractDeclaredNamespace.php

```diff
@@ -12,7 +12,7 @@ class ExtractDeclaredNamespace extends NodeVisitorAbstract
 
     public function enterNode(Node $node)
     {
-        if ($node instanceof Node\Stmt\ClassLike && $node->namespacedName !== null) {
+        if ($node instanceof Node\Stmt\ClassLike && isset($node->namespacedName) && $node->namespacedName  !== null) {
```

Comment:
Anonymous classes has no `namespacedName` field. So comparing it with null ends up with "Undefined field" notice